CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## v1.3.0 (2023-10-10)
+### Changed
+ - Dependencies have been updated: testify, logrus, govmomi
+ - The way snapshot sizes are computed has been refactored in order to take into account delta disks
```

Comment:
was this a bugfix?

Reply:
it depends on how you see it, it was initially filed as FR. I was considering it a bug as well since the delta disks were essentials. If you want I can change it, no problem 😄

Reply:
same, just thinking about how this value will change , perhaps it has more visibility as a bugfix